### PR TITLE
[MDS-5431] Create new or historical record from one window

### PR DESCRIPTION
**Pull request by isuru-aot**
## Objective 

[MDS-5431](https://bcmines.atlassian.net/browse/MDS-5431)

Why are you making this change? Provide a short explanation and/or screenshots_
Screenshot 2023-10-06 143913

Screenshot 2023-10-06 143856

Screenshot 2023-10-06 143924
Update: Typo in the screenshot is corrected.

### CODE REVIEW

**Review by henryoforeh-dev — DISMISSED** [review]
Looks good!

File: services/core-web/src/tests/components/Forms/ExplosivesPermit/__snapshots__/ExplosivesPermitForm.spec.js.snap [hunk 1]
```diff
@@ -1,258 +1,135 @@
...
-              }
-            />
-          </FormItem>
-        </Col>
-        <Col
-          span={12}
-        >
-          <FormItem
-            hasFeedback={false}
+            Add an existing explosive storage and Use permit
```
Comment (taraepp) [inline services/core-web/src/tests/components/Forms/ExplosivesPermit/__snapshots__/ExplosivesPermitForm.spec.js.snap]: Only "Use" is capitalized here.

File: services/core-web/src/components/Forms/ExplosivesPermit/ExplosivesPermitForm.tsx [hunk 2]
```diff
@@ -117,13 +118,118 @@ export const ExplosivesPermitForm: FC<ExplosivesPermitFormProps &
...
+              conditions, such as the dates, amount of explosives.</Typography.Text>
+          </li>
+        </ul>
+      </Typography.Paragraph>
+    </div>
+  );
+
+  const amendDescriptionListElement = (
+    <div>
+      To make changes to an existing explosive storage and use permit,\u2022 follow these steps:
```
Comment (taraepp) [inline services/core-web/src/components/Forms/ExplosivesPermit/ExplosivesPermitForm.tsx]: in the screenshot the u2022 character is not rendering as intended

**Review by taraepp — CHANGES_REQUESTED** [review]
Couple minor issues with text display but once those are dealt with LGTM. 👍

File: services/core-web/src/tests/components/Forms/ExplosivesPermit/__snapshots__/ExplosivesPermitForm.spec.js.snap (hunk 1 above)
Comment (isuru-aot) [reply services/core-web/src/tests/components/Forms/ExplosivesPermit/__snapshots__/ExplosivesPermitForm.spec.js.snap]: corrected.

File: services/core-web/src/components/Forms/ExplosivesPermit/ExplosivesPermitForm.tsx (hunk 2 above)
Comment (isuru-aot) [reply services/core-web/src/components/Forms/ExplosivesPermit/ExplosivesPermitForm.tsx]: Corrected. 
Screenshot 2023-10-10 100632

**Review by taraepp — DISMISSED** [review]
Wow these tests are slow today. I'm assuming they're going to pass, though. :approved:

**Review by henryoforeh-dev — DISMISSED** [review]

**Review by taraepp — APPROVED** [review]

**Review by matbusby-fw — APPROVED** [review]

**Review by henryoforeh-dev — APPROVED** [review]

**Pull request merged**